cisco_umbrella_reporting/help.md

```diff
@@ -0,0 +1,124 @@
+# Description
+
+This plugin utilizes Cisco Umbrella to get the most complete view of the relationships and evolution of internet domains, IP addresses, and autonomous systems to pinpoint attackers infrastructures and predict future threats
+
+# Key Features
+
+* Most Recent Requests
+* Security Activity Report
+
+# Requirements
+
+* Requires an API Key and Secret from Cisco Umbrella
+* Requires Cisco Umbrella organization ID
+
+# Documentation
+
+## Setup
+
+The connection configuration accepts the following parameters:
+
+|Name|Type|Default|Required|Description|Enum|Example|
+|----|----|-------|--------|-----------|----|-------|
+|api_key|credential_secret_key|None|True|Cisco Umbrella API key|None|9de5069c5afe602b2ea0a04b66beb2c0|
+|api_secret|credential_secret_key|None|True|Cisco Umbrella API secret key|None|9de5069c5afe602b2ea0a04b66beb2c0|
+|organization_id|string|None|True|ID of your Cisco Umbrella organization|None|2961483|
+
+Example input:
+
+```
+{
+  "api_key": "9de5069c5afe602b2ea0a04b66beb2c0",
+  "api_secret": "9de5069c5afe602b2ea0a04b66beb2c0",
+  "organization_id": 2961483
+}
+```
+
+## Technical Details
+
+### Actions
+
+#### Get Domain Visits
+
+This action is used to get a list of computers that visited a domain for the last 24 hours.
```

Comment:
```suggestion
This action is used to get a list of computers that visited a domain for the last 24 hours and up to 500 results.
It accepts the following address types: domains, IPs, and URLs as input. URLs will automatically be stripped down to the domain since the Umbrella API does not accept URLs. If no add address is provided, then the action will return activities for all domains in the organization.
```